less/components/nav.less

```diff
@@ -39,6 +53,8 @@
       height: 65px;
       display: flex;
       align-items: center;
+      justify-content: flex-start;
+      flex-direction: row;
```

Comment:
Both of these ^^

If I remove them locally, everything seems to look the same.